lib/luahid.c

```diff
@@ -109,13 +112,31 @@ static inline void luahid_pushdevid(lua_State *L, int idx, const struct hid_devi
 	luahid_setfield(L, -1, device_id, driver_data);
 }
 
+static inline void luahid_pushhdev(lua_State *L, struct hid_device *hdev)
+{
+	lua_newtable(L);
```

Comment:
what happens if this (or any other call below) raises an error? (you propably will lua_pcall)